splunk-on-ubuntu/azuredeploy.json

```diff
@@ -235,31 +238,67 @@
       "clusterIndexerVmSize": "[parameters('clusterIndexerVmSize')]",
       "diskSize": 1023,
       "standaloneTemplateUrl": "[variables('emptyTemplateUrl')]",
-      "clusterMasterTemplateUrl": "[concat(variables('templateBaseUrl'), 'nested/', 'clustermaster-resources.json')]",
-      "clusterSearchheadTemplateUrl": "[concat(variables('templateBaseUrl'), 'nested/', 'clustersearchhead-resources.json')]",
-      "clusterPeersTemplateUrl": "[concat(variables('templateBaseUrl'), 'nested/', 'clusterpeers-resources.json')]",
+      "clusterMasterTemplateUrl": "[concat(variables('templateNestedUrl'), 'clustermaster-resources.json')]",
+      "clusterSearchheadTemplateUrl": "[concat(variables('templateNestedUrl'), 'clustersearchhead-resources.json')]",
+      "clusterPeersTemplateUrl": "[concat(variables('templateNestedUrl'), 'clusterpeers-resources.json')]",
       "clusterPeersCount": "[parameters('clusterIndexerVmCount')]",
       "availabilitySetCount": 3,
       "publicIPAddress": {
         "name": "[parameters('publicIPName')]",
         "domainNamePrefix": "[parameters('domainNamePrefix')]",
         "count": "[add(2, parameters('clusterIndexerVmCount'))]",
         "map": [
-          "", "-cm",
-          "-cp0", "-cp1", "-cp2", "-cp3", "-cp4",
-          "-cp5", "-cp6", "-cp7", "-cp8", "-cp9",
-          "-cp10", "-cp11", "-cp12", "-cp13", "-cp14",
-          "-cp15", "-cp16", "-cp17", "-cp18", "-cp19"
+          "",
+          "-cm",
+          "-cp0",
+          "-cp1",
+          "-cp2",
+          "-cp3",
+          "-cp4",
+          "-cp5",
+          "-cp6",
+          "-cp7",
+          "-cp8",
+          "-cp9",
+          "-cp10",
+          "-cp11",
+          "-cp12",
+          "-cp13",
+          "-cp14",
+          "-cp15",
+          "-cp16",
+          "-cp17",
+          "-cp18",
+          "-cp19"
         ]
       },
       "storage": {
         "name": "[variables('storageAccountNamePrefix')]",
         "type": "[parameters('storageAccountType')]",
         "count": "[div(sub(add(add(2, parameters('clusterIndexerVmCount')), variables('nodesPerStorageAccount')), 1), variables('nodesPerStorageAccount'))]",
         "map": [
-          0, 1, 1, 0, 1, 1,
-          2, 2, 2, 2, 3, 3, 3, 3,
-          4, 4, 4, 4, 5, 5, 5, 5
+          0,
+          1,
+          1,
+          0,
+          1,
+          1,
+          2,
+          2,
+          2,
+          2,
+          3,
+          3,
+          3,
+          3,
+          4,
+          4,
+          4,
+          4,
+          5,
+          5,
+          5,
+          5
```

Comment:
Any reason for this change?
The original map layout was intentional to show the storage account allocation per vm.